src/CustomStyleCommand.js

```diff
@@ -808,7 +813,7 @@ function hasMismatchHeirarchy(
     }
     if (levelDiff > 0) {
       if (selectedNodes.length === 1) {
-        nodesBeforeSelection.reverse();
+        // nodesBeforeSelection.reverse();
```

Comment:
Don't leave commented out code.  Keeping history is what GitHub is for.
Don't leave empty conditions.